verl/workers/tpu_workers.py

```diff
@@ -101,26 +101,13 @@ def __init__(self, config: DictConfig, role: str):
         self.config = config
         self.device_name = get_device_name()
 
-        import torch.distributed
-
-        if not torch.distributed.is_initialized():
-            rank = int(os.environ.get("RANK", 0))
-            world_size = int(os.environ.get("WORLD_SIZE", 1))
-            torch.distributed.init_process_group(backend="cpu:gloo,cuda:nccl" if is_cuda_available else "cpu:gloo,npu:hccl", rank=rank, world_size=world_size)
+        rank = int (os.environ.get("RANK", 0))
+        world_size = int(os.environ.get("WORLD_SIZE", 1))
 
-        # build device mesh for FSDP
-        world_size = torch.distributed.get_world_size()
         # TODO(sgm): support FSDP hybrid shard for larger model
-        self.device_mesh = create_device_mesh(world_size=world_size, fsdp_size=self.config.actor.fsdp_config.fsdp_size)
-
-        # build device mesh for Ulysses Sequence Parallel
-        self.ulysses_device_mesh = None
-        self.ulysses_sequence_parallel_size = self.config.actor.get("ulysses_sequence_parallel_size", 1)
-        dp = world_size // self.ulysses_sequence_parallel_size
-        if self.ulysses_sequence_parallel_size > 1:
-            self.ulysses_device_mesh = init_device_mesh(self.device_name, mesh_shape=(dp, self.ulysses_sequence_parallel_size), mesh_dim_names=["dp", "sp"])
+        self.device_mesh_size = world_size
+        self.ulysses_sequence_parallel_size = 1
```

Comment:
Please add a TODO to remove this in a future PR since we don't have any plan to support sequence parallel